<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoShow</class>
 <widget class="QWidget" name="PhotoShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Show</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Comment:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLineEdit" name="dateLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="locLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="commentLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Display Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RSComboBox" name="sizeComboBox">
         <item>
          <property name="text">
           <string>320 x 320</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>640 x 640</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Full Size</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Play Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="RSComboBox" name="speedComboBox">
         <item>
          <property name="text">
           <string>1 Sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 Sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 Sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 Sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 Sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 Min</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>Edit Photo Details</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save Photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="photoLabel">
       <property name="text">
        <string>No Photo Selected</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>Back</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>Play </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepButton">
         <property name="text">
          <string>Forward</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RSComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/common/RSComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
